### ISSUE TITLE
Move batch size parameter to settings

### ISSUE THREAD

**Issue by tobigithub** [josharnoldjosh/massspectra#11]
The definition of **batch size** should go to settings.py
````
        for i in range(0,num_models_to_average):
            time.sleep(1)
            model = nn.baseline_model(input_dim_val=X_train.shape[1], output_dim_val=output_dim)
            result = model.fit(X_train, y_train,                                
                               batch_size=20, epochs=settings.epoch_amount, 
                               validation_data=(X_test, y_test), 
                               callbacks=[earlystopping,checkpoint, tensorboard])
            models.append(model)
            results.append(result)   
````

See here batch size also influences outcomes, smaller batch size influences the overall similarity,
however some compounds are worse, some are better.  Smaller batch size = better accuracy = slower training per epoch.

https://stats.stackexchange.com/questions/164876/tradeoff-batch-size-vs-number-of-iterations-to-train-a-neural-network

````
Batch size 40
Model 1  score: 562.282434484
Model 1  acc: 0.725274722655

Test score: 562.282434484
Average test accuracy: 0.725274722655
NN total train time: 0h:03m:07s
Average cosine similarity: 0.940347806286
Script finished.
````
----------------------------------------------------------------------------
````
Batch size 20

Model 1  score: 579.893168355
Model 1  acc: 0.747252738083

Test score: 579.893168355
Average test accuracy: 0.747252738083
NN total train time: 0h:04m:15s
Average cosine similarity: 0.942715065631
Script finished.
```

**Comment by josharnoldjosh** [comment]
Fixed in latest commit.

**Comment by tobigithub** [comment]
Great.
